### PR TITLE
add exe 13 file

### PR DESCRIPTION
**Pull request by Reyes-PaulAngelo**
add my initial exe 13 files

### CODE REVIEW

File: Reyes_PaulAngelo/index.html [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+      </p>
+
+      <div id="comments-container"></div>
+      <button id="sort-asc">Sort by Date (Asc)</button>
+      <button id="sort-desc">Sort by Date (Desc)</button>
+    </div>
+
+    <div id="comments">
+      <h2>Make Comments</h2>
+      <form id="comment-form">
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/index.html]: IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: Reyes_PaulAngelo/index.html [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        <input type="text" id="comment-name" placeholder="Your Name" />
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/index.html]: IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: Reyes_PaulAngelo/index.html [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+        <textarea id="comment-text" placeholder="Your Comment"></textarea>
+        <button type="button" id="submit-comment" disabled>Comment</button>
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/index.html]: IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: Reyes_PaulAngelo/index.html [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+      </p>
+      <p>Name: Daisy Cangayda Morcoso</p>
+      <p>
+        Comment: It's impressive how your goals are aligned with your values 
+        and aspirations, guiding you towards a fulfilling path.
+      </p>
+
+      <div id="comments-container"></div>
+      <button id="sort-asc">Sort by Date (Asc)</button>
+      <button id="sort-desc">Sort by Date (Desc)</button>
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/index.html]: IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: Reyes_PaulAngelo/script.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const commentNameInput = document.getElementById("comment-name");
+  const commentTextInput = document.getElementById("comment-text");
+  const submitCommentButton = document.getElementById("submit-comment");
+  const commentForm = document.getElementById("comment-form");
+  const commentsContainer = document.getElementById("comments-container");
+  const sortAscButton = document.getElementById("sort-asc");
+  const sortDescButton = document.getElementById("sort-desc");
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/script.js]: IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: Reyes_PaulAngelo/script.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+    addComment(name, text, new Date());
+    commentForm.reset();
+    checkCommentValidity();
+  });
+
+  function addComment(name, text, date) {
+    const commentElement = document.createElement("p");
+    commentElement.innerHTML = `Name: ${name}<br>Comment: 
+    ${text}<br><small>${date.toLocaleString()}</small>`;
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/script.js]: Do not use the `<br>` element. Apply proper styling instead.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g28d48e53e08_0_139

File: Reyes_PaulAngelo/script.js [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+    renderComments();
+  }
+
+  function renderComments() {
+    commentsContainer.innerHTML = "";
+    comments.forEach((comment) => {
+      const commentElement = document.createElement("p");
+      commentElement.innerHTML = `Name: ${comment.name}<br>Comment: ${
+        comment.text
+      }<br><small>${comment.date.toLocaleString()}</small>`;
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/script.js]: Do not use the `<br>` element. Apply proper styling instead.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g28d48e53e08_0_139

File: Reyes_PaulAngelo/style.css [hunk 8]
```diff
@@ -0,0 +1,236 @@
...
+  margin-bottom: 20px;
+}
+
+#comments {
+  border: 2px black solid;
+  margin-bottom: 20px;
+  margin: 1;
+}
+
+#form{
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/style.css]: Observe proper indentation.

File: Reyes_PaulAngelo/works/IT.html [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+        <h2>Skills Needed</h2>
+        <p>
+            People require a combination of skills in order to 
+            succeed in the IT industry of technical proficiency, 
+            aptitude for solving problems, and effective 
+            communication abilities. Flexibility, originality, 
+            and a readiness to keep learning and evolving 
+            are also crucial to new technologies.            
+        </p>
+        </div>
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/works/IT.html]: Observe proper indentation.

**Review by Villa-VillaDiana — CHANGES_REQUESTED** [review]

File: Reyes_PaulAngelo/script.js [hunk 10]
```diff
@@ -50,9 +50,9 @@ document.addEventListener("DOMContentLoaded", function () {
     commentsContainer.innerHTML = "";
     comments.forEach((comment) => {
       const commentElement = document.createElement("p");
-      commentElement.innerHTML = `Name: ${comment.name}<br>Comment: ${
+      commentElement.innerHTML = `Name: ${comment.name}<p>Comment: ${
         comment.text
-      }<br><small>${comment.date.toLocaleString()}</small>`;
+      }<p><small>${comment.date.toLocaleString()}</small>`;
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/script.js]: I suggest following this format for readability
```
      commentElement.innerHTML = `
        `Name: ${comment.name}
        <p>Comment: ${comment.text}
        <p><small>${comment.date.toLocaleString()}</small>`;
```

File: Reyes_PaulAngelo/script.js [hunk 11]
```diff
@@ -34,8 +34,8 @@ document.addEventListener("DOMContentLoaded", function () {
 
   function addComment(name, text, date) {
     const commentElement = document.createElement("p");
-    commentElement.innerHTML = `Name: ${name}<br>Comment: 
-    ${text}<br><small>${date.toLocaleString()}</small>`;
+    commentElement.innerHTML = `Name: ${name}<p>Comment: 
+    ${text}<p><small>${date.toLocaleString()}</small>`;
```
Comment (Villa-VillaDiana) [inline Reyes_PaulAngelo/script.js]: I suggest following this format for readability

```
    commentElement.innerHTML = 
      `Name: ${name}
      <p>Comment: ${text}<p><small>${date.toLocaleString()}</small>`;
```

**Review by Villa-VillaDiana — CHANGES_REQUESTED** [review]

**Review by Villa-VillaDiana — APPROVED** [review]
Looks good to me, you can now merge this :))

**Comment by Reyes-PaulAngelo** [comment]
Thank you po <3

**Pull request merged**